src/main/java/io/github/oliviercailloux/y2018/jconfs/Conference.java

```diff
@@ -91,12 +93,13 @@ public Double getFeeRegistration() {
 	/**
 	 * This is a setter which set the fee of registration
 	 * @param registrationFee
+	 * 			not <code>null</code>
 	 */
 	public void setFeeRegistration(Double registrationFee) {
```

Comment:
Si vous vouliez rendre ce paramètre obligatoire, il devrait être de type primitif `double`, pas `Double` (et retirer `not null` qui deviendrait inutile).